wrappers/python/simtk/openmm/app/charmmparameterset.py

```diff
@@ -481,12 +515,16 @@ def readParameterFile(self, pfile):
         # instances. In order for this to work, all keys in nonbonded_types
         # must be in the self.atom_types_str dict. Raise a RuntimeError if this
         # is not satisfied
-        try:
-            for key in nonbonded_types:
+        for key in nonbonded_types:
+            try:
                 self.atom_types_str[key].set_lj_params(*nonbonded_types[key])
-        except KeyError:
-            raise RuntimeError('Atom type %s not present in AtomType list' %
+            except KeyError:
+                if( permissive ):
+                    print( " Atom type " + key + " unknown" )
```

Comment:
Turn this into a `warnings.warn` to avoid printing from the API.  This allows it to be intercepted via loggers and otherwise handled/silenced (and sends it to stderr instead of stdout).